com.palantir.typescript/src/com/palantir/typescript/services/language/QuickInfo.java

```diff
@@ -79,6 +82,14 @@ public String getDisplayText() {
         return this.documentation;
     }
 
+    public List<JSDocTagInfo> getTags() {
+        return this.tags;
+    }
+
+    public String getTagsText() {
```

Comment:
I was unsure if you want to add the text in the Over. At least the method is there.